src/main/scala/ru/gaptrakhmanov/telegram/helper/commands/wordgame/WordInitService.scala

```diff
@@ -0,0 +1,43 @@
+package ru.gaptrakhmanov.telegram.helper.commands.wordgame
+
+import cats.effect.Sync
+import cats.implicits._
+import ru.gaptrakhmanov.telegram.helper.commands.wordgame.dto.{User, WordSet}
+import ru.gaptrakhmanov.telegram.helper.commands.wordgame.state.WordGameState
+import ru.gaptrakhmanov.telegram.helper.util.UrlUtils
+
+object WordInitService {
+
```

Comment:
One advice on this implementation is that when we interact with different side effects in this way, the different operation are not atomic, meaning if we receive two multiple requests for the same user on `getWords` per example it is possible that the two requests will think that the state is empty and both of them will try to `getWordsFromUrl`. To avoid this scenario in the case that we just want to update the state and return something without doing side effects inside we can use the modify and add the logic inside, what mean that probably we will always need to call the  `getWordsFromUrl` before to try to modify.